workflows/Lookup_Vulnerable_Hosts_From_Slack/workflow.spec.yaml

```
@@ -3,13 +3,13 @@ products: ["insightconnect", "insightvm"]
 name: Lookup_Vulnerable_Hosts_From_Slack
 title: Lookup Vulnerable Hosts from Slack
 description: Lookup vulnerability details, including a summary of the vulnerability, the affected software package, the remediation or mitigation step, and a list of all affected hosts in the user's environment.
-version: 1.1.0
+version: 2.0.0
```

```suggestion
version: 1.2.0
```
Mismatch between changelog and this version